packages/node-opentelemetry/README.md

```diff
@@ -223,3 +222,19 @@ process.on('SIGTERM', async () => {
   process.exit();
 });
 ```
+
+### GCP Cloud Function Event Handler
+
+If your application runs on GCP (Google Cloud Platform) Cloud Functions, you can use the `registerGCPCloudFunctionEventHandler` function to automatically instrument your function.
```

Comment:
I think we can add more context:

Currently the event handler will ensure traces gets propagated through PubSub as long as the publisher has `enableOpenTelemetryTracing: true` (see [example](https://github.com/googleapis/nodejs-pubsub/blob/730e9dc0ab20a5f02508a82310d58b0da1f54330/samples/openTelemetryTracing.js#L86)).